package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "roles-subgraph",
+  "version": "1.0.0",
+  "author": "orium.network",
+  "license": "MIT",
+  "description": "A set of subgraphs (one for each network) for Orium Network dapps",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "graph-auth": "graph auth --product hosted-service",
+    "codegen-goerli": "cp addresses-goerli.ts src/utils/addresses.ts && cp subgraph-goerli.yaml subgraph.yaml && graph codegen subgraph-goerli.yaml",
+    "build-goerli": "graph build subgraph-goerli.yaml",
+    "deploy-goerli": "graph deploy --node https://api.thegraph.com/deploy/ oriummgnt/orium-goerli",
```

We should change the subgraph name that we going to deploy it. `orium-goerli` is already reserved for another subgraph. Ask @ernanirst for the respective subgraph name for roles-registry on goerli.
 
```suggestion
    "deploy-goerli": "graph deploy --node https://api.thegraph.com/deploy/ oriummgnt/roles-registry-goerli",
```